test/image/mocks/geo_choropleth-legend.json

```diff
@@ -0,0 +1,67 @@
+{
+    "data": [
+        {
+            "type": "choropleth",
+            "name": "Canada",
+            "locations": ["CAN"],
+            "z": [1],
+            "colorscale": [[0, "blue"], [1, "blue"]],
```

Comment:
cc https://github.com/plotly/plotly.js/issues/3468 and https://github.com/plotly/plotly.js/issues/1747